### PR TITLE
Rewrote some remaining shell scripts in Python.

### PR DESCRIPTION
**Pull request by craftnut**
gud :+1:

### CODE REVIEW

File: Prefix_CopyAll.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import os
+from distutils.dir_util import copy_tree
+
+cp_from = input("Folder to copy: ")
+cp_to = input("Folder to copy to: ")
+
+copy_tree(cp_from, cp_to)
```
Comment (zeptofine) [inline Prefix_CopyAll.py:7]: not all the original Prefix_CopyAll did

File: Difference.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            )
+            
+        else:
+            (
+                ffmpeg
+                .input(f"{dir_in}/{in_list[file]}")
+                .filter('scale', res)
+                .output(f"{dir_out}/{in_list[file]}.{file_type}")
+                .run()
+            )
```
Comment (zeptofine) [inline Difference.py:32]: this could be done with cv2 and numpy. likely much faster

File: Difference.py
Comment (zeptofine) [inline Difference.py:1]: I swear difference did more?? I must've commented it out

File: Difference.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        if "mp4" or "mov" or "gif" in in_list[file]:
+            os.mkdir(f"{dir_in}/{in_list[file]}split")
+            split_out = f"{dir_in}/{in_list[file]}split"
+            (
+            ffmpeg
+            .input(f"{dir_in}/{in_list[file]}")
+            .filter('scale', res)
+            .output(f"{split_out}/{in_list[file]}.{file_type}")
+            .run()
+            )
```
Comment (zeptofine) [inline Difference.py:23]: difference was made for images

File: Difference.py
Comment (craftnut) [reply Difference.py:1]: expanded functionality

**Pull request merged**